core/services/vrf/listener_v2.go

```diff
@@ -400,8 +400,17 @@ func (lsn *listenerV2) processPendingVRFRequests(ctx context.Context) {
 		sub, err := lsn.coordinator.GetSubscription(&bind.CallOpts{
 			Context: ctx,
 		}, subID)
+
+		if err != nil && strings.Contains(err.Error(), "execution reverted") {
+			lsn.l.Warnw("Subscription not found", "subID", subID, "err", err)
+			for _, req := range reqs {
+				processed[req.req.RequestId.String()] = struct{}{}
+			}
+			continue
+		}
+
 		if err != nil {
```

Comment:
nit: Doesn't change functionality but I like to do an "else if" rather than an "if" here so it's visually clear that both checks are part of the same error handling logic

Reply:
sounds good. updated